homeassistant/components/notify/gntp.py

```diff
@@ -0,0 +1,58 @@
+"""
+GNTP (aka Growl) notification service.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/notify.gntp/
+"""
+import logging
+import os
+
+from homeassistant.components.notify import (
+    ATTR_TITLE, BaseNotificationService)
+
+REQUIREMENTS = ['gntp==1.0.3']
+
+_LOGGER = logging.getLogger(__name__)
+
+_GNTP_LOGGER = logging.getLogger('gntp')
+_GNTP_LOGGER.setLevel(logging.ERROR)
+
+
+def get_service(hass, config):
+    """Get the GNTP notification service."""
+    if config.get('app_icon') is None:
+        icon_file = os.path.join(os.path.dirname(__file__), "..", "frontend",
+                                 "www_static", "favicon-192x192.png")
+        app_icon = open(icon_file, 'rb').read()
+    else:
+        app_icon = config.get('app_icon')
```

Comment:
If no app_icon given, you will read in the file contents. Otherwise use the config value. Do you expect the people to pass the content of an image in the config?